### DISCUSSION TITLE
SSH Ramdisk iOS Build ID Selection.

### DISCUSSION THREAD

**Discussion by userse31** [LukeZGD/Legacy-iOS-Kit#334] [Ideas]
For the iPhone 5c, the script defaults to using ios build id "11D257" (ios 7.1.2) for the SSH ramdisk.

However, iOS 9 and 10 change the way the data partition is handled, meaning the ramdisk MUST be atleast iOS 9 if iOS 9 or greater is installed on said phone if one wants to manipulate the data partition (eg, data backups).

It would be nice if the script prompted for an iOS build number to use when building an SSH ramdisk, like the "just boot" tethered boot option.

Although, there does seem to be weird issues with port 22 being "banned" on these newer ios versions.

It can be changed by modifying the "device_type" case block around line 3130, but, well, that isn't convenient.

**Comment by LukeZGD** [comment]
Relevant issue: #325

Mounting /var for iOS 9/10 sshrd is now fixed in v23.10.12 (e6f0916). The script will now prompt for build version. To mount /var for iOS 9/10, create an iOS 9 or newer ramdisk
